projects/lf-documentation/src/app/example-usage-in-angular/example-usage-in-angular.component.html

```diff
@@ -41,6 +41,15 @@ <h3 id="cdn-example-angular"> Example: <code>lf-checklist</code></h3>
   >"https://cdn.jsdelivr.net/npm/@laserfiche/lf-ui-components@NPM_VERSION/cdn/indigo-pink.css"</span><span class="code-html-attribute-name"> rel</span><span
   >=</span><span class="code-html-attribute-value">"stylesheet" </span><span class="code-html-tagname">/&gt;</span>
 </pre>
+</li>
+<li>
+  <p  class="documentation-text">
+    Add Zone.js if not already imported
+  </p>
```

Comment:
since this is angular, did you test it will just having it in polyfills.ts?
Bc that's where most angular projects will already have it